azure/resource_groups/app/parameters/development.template.json

```diff
@@ -181,6 +181,9 @@
     "ENVIRONMENT_NAME": {
       "value": "development"
     },
+    "WORKER_COUNT": {
+      "value": 1
```

Comment:
Do you think we should go for parity with production here? It should help us pick up on any issues sooner

Reply:
Argh, my brain still treats `development` as "to develop in" not "staging". Fixed!